### PR TITLE
Merge better-locale-selection to main

### PR DESCRIPTION
**Pull request by Oakchris1955**
The reason I am not directly commiting this is because I want @moritz-t-w 's opinion about what I did

### CODE REVIEW

**Review by moritztim — APPROVED** [review]
seems like a good solution

**Comment by Oakchris1955** [comment]
Imma check it tomorrow too and then merge

File: src/main.rs [hunk 1]
```diff
@@ -121,6 +146,10 @@ fn get_user_input(msg: &str, secret_number: u32, locale: &Localization) -> u32 {
 			}
 		}
 
+		if commands::check_for_valid_command_name(&user_input) {
```
Comment (moritztim) [inline src/main.rs]: Just saw this, I think this could be called check_command_name() or validate_command_name()

**Review by moritztim — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (Oakchris1955) [reply src/main.rs]: Well, actually, I accidentally commited that. I would commit it on another branch. Imma fix it and when commit it, name it `validate_command_name`

**Review by moritztim — APPROVED** [review]
Nice

**Pull request merged**